WordPress/Classes/ViewRelated/NUX/SiteInfoHeaderView.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+/// A subclass of BlogDetailHeaderView styled for use in the login flow.
+///
+class SiteInfoHeaderView: BlogDetailHeaderView {
+
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        configureStyles()
+    }
+
+
+    func configureStyles() {
+
+        titleLabel.font = WPFontManager.systemSemiBoldFont(ofSize: 15.0)
+        titleLabel.sizeToFit()
+        titleLabel.textColor = WPStyleGuide.darkGrey()
+
+        subtitleLabel.font = UIFont.preferredFont(forTextStyle: .footnote)
+        subtitleLabel.sizeToFit()
```

Comment:
I'm not certain the `sizeToFit` calls are necessary. Not a big deal, but can you double check?